packages/vats/src/repl.js

```diff
@@ -5,6 +5,21 @@ import { getInterfaceOf, Remotable, Far } from '@agoric/marshal';
 import { Nat } from '@agoric/nat';
 import makeUIAgentMakers from './ui-agent';
 
+const UNJSONABLES = new Map([
+  [NaN, 'NaN'],
+  [Infinity, 'Infinity'],
+  [-Infinity, '-Infinity'],
+  [undefined, 'undefined'],
+]);
+
+for (const [name, { value }] of Object.entries(
+  Object.getOwnPropertyDescriptors(Symbol),
+)) {
+  if (typeof value === 'symbol' && typeof name === 'string') {
+    UNJSONABLES.set(value, `Symbol(Symbol.${name})`);
+  }
+}
+
 // A REPL-specific JSON stringify.
```

Comment:
This comment contributed to my misunderstanding that this is an *adjective* JSON stringify, for some adjective.

Reply:
I'll clear that up.

Reply:
Renamed `stringify` to `dump(obj, spaces = undefined)`.